app/controllers/admin/groups_controller.rb

```diff
@@ -26,7 +26,10 @@ def edit
   end
 
   def update
-    if @group.update_attributes(group_params)
+    filtered_group_params = group_params
+    filtered_group_params.delete(:group_memberships_attributes)
```

Comment:
This could be single-lined;

`filtered_group_params = group_params.except(:group_memberships_attributes)`

See http://api.rubyonrails.org/classes/Hash.html#method-i-except